documentation/docs/pages/Parameter-choices.md

```diff
@@ -0,0 +1,249 @@
+
+#### Internal tidal mixing
+`INT_TIDE_DISSIPATION = True` turns on the internal tidal mixing. It activates the parameterisation of internal tidal energy dissipation. We use `INT_TIDE_PROFILE = "POLZIN_09"`, following the tidal energy from [@polzin2009abyssal] stretched exponential profile rather than the default St. Laurent exponential. We also set `READ_TIDEAMP = True` with a `tideamp.nc` file and roughness data (`H2_FILE = "bottom_roughness.nc"`). Both files were generated using updated bottom roughness calculated from [`SYNBATH`](https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2021EA002069) and tidal velocities from [`TPXO10`](https://www.tpxo.net/global/tpxo10), processed via through [om3-scripts/external_tidal_generation](https://github.com/ACCESS-NRI/om3-scripts/tree/main/external_tidal_generation). This indicates the model reads spatial maps of tidal velocity amplitude and topographic roughness to inform where internal tides dissipate energy. By doing so, the vertical diffusivity can be enhanced in regions of rough bathymetry and high tidal speeds. `TKE_ITIDE_MAX = 0.1` limits the energy per area can be injected as mixing. Overall, turning on the internal tidal mixing is crucial for simulating the deep ocean stratification and circulation. 
+
+#### Interior background mixing
+For the ocean interior background mixing, we follow the approach from [GFDL OM5 configuration](https://github.com/NOAA-GFDL/MOM6-examples/blob/3c1de3512e2200bfc10d9e5150715c9df76dbd30/ice_ocean_SIS2/Baltic_OM5_025/MOM_parameter_doc.all#L2004) of using a weak constant background diapycnal diffusivity (`KD = 1.5E-05`) for diapycnal mixing. A floor `KD_MIN = 2.0e-6` is also applied, so it won’t go below 2e-6 anywhere, ensuring numerical stability. We enable `DOUBLE_DIFFUSION = True`, which enhances vertical mixing for salt-fingering. Henyey-type internal wave scaling is set through `HENYEY_IGW_BACKGROUND = True`. The parameters `HENYEY_N0_2OMEGA = 20.0` and `HENYEY_MAX_LAT = 73.0` are kept at default. At the same time, to prevent unbounded growth of shear-based or convective mixing, we cap the total diffusivity increment from TKE-based schemes with `KD_MAX = 0.1`. This is a large upper bound that would only be triggered in extremely unstable cases.
+
+The bottom drag is quadratic with coefficient `CDRAG = 0.003`, which is a typical value from ocean observations. `BOTTOMDRAGLAW = True` with `LINEAR_DRAG = False` means a quadratic bottom drag law rather than a linear damping. The thickness of the bottom boundary layer is set to `HBBL = 10.0`.
```

Comment:
```suggestion
The bottom drag is quadratic with coefficient `CDRAG = 0.003`, which is a typical value from ocean observations. `BOTTOMDRAGLAW = True` with `LINEAR_DRAG = False` means a quadratic bottom drag law rather than a linear damping. The thickness of the bottom boundary layer is set to `HBBL = 10.0` m.
```